src/main/java/at/hannibal2/skyhanni/data/HypixelData.kt

```diff
@@ -183,8 +182,11 @@ class HypixelData {
             ScoreboardData.sidebarLinesFormatted.matchFirst(scoreboardVisitingAmoutPattern) {
                 return group("maxamount").toInt()
             }
-            if (IslandType.CRYSTAL_HOLLOWS.isInIsland()) return 24
-            return if (serverId?.startsWith("mega") == true) 80 else 26
+            val islandSpecific = mapOf(
```

Comment:
please use a when here. Instead of what ever this is. The null case is just the else branch of the when